.github/workflows/merge-rules.yml

```
@@ -0,0 +1,37 @@
+name: 'Pull Request Check'
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+
+jobs:
+  check-commit-message:
+    name: Check Commit Message
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get Commit Message
+        run: |
+          nlines=-1
+          for (( N=1; N<=${{github.event.pull_request.commits}}; N++ ))
+          do
+            prev_lines=$(($nlines+1))
+            nlines=0
+            while IFS= read -r line
+            do
+              nlines=$((nlines+1))
+              if (( $nlines == $prev_lines+1 )); then
+                if ! [[ "$line" =~ ^\[#([0-9])+\]\ .+$|^(\[NO-ISSUE\]\ ).+$ ]]; then
```

Are the round brackets required?
```suggestion
                if ! [[ "$line" =~ ^\[#([0-9])+\]\ .+$|^\[NO-ISSUE\]\ .+$ ]]; then
```

---

Indeed they are not necessary. good catch!